.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,3 +39,4 @@ Delete the checkbox if it doesn't apply/isn't necessary.
 - [ ] Update documentation
 - [ ] Link all relevant tickets
 - [ ] Move the tickets on the board accordingly
+- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))
```

Comment:
I think we can use https://github.com/demos-europe/demosplan-documentation/blob/main/development/guidelines-conventions/coding-styleguides/twig_html.md#guideline-for-naming-cypress-hooks as a link, to give people outside our  vpn the possibility to read the docs.

Reply:
I would leave it like so. For our convenience.